config/vufind/config.ini

```diff
@@ -1694,10 +1694,18 @@ limit   = 3
 phrase = false
 ; Offer expansions on terms as well as basic replacements
 expand  = true
-; Turning on 'simple' spell checking will improve performance,
-;  by ignoring the more complicated 'shingle' (mini phrases)
-;  based dictionary.
-simple = false
+; Set the list of spellcheck dictionaries used in a Solr query. The
+; available dictionaries are defined in solrconfig.xml.
+; To replicate the legacy 'simple' option, to improve performance by ignoring
+; the more complicated 'shingle' (mini phrases) based dictionary, disable
+; 'default' to use only 'basicSpell'.
+; To use Solr's DirectSolrSpellChecker dictionary, enable 'direct' and
```

Comment:
Have you tested what happens if you enable all the dictionaries at once? Just curious if that's something we need to potentially worry about for any reason.

Reply:
Sort of but not really.  I never ran spellcheck.build=true, so the default and basicSpell dictionaries do nothing, and enabling all three just effectively enables direct.  Nothing breaks.  I have not tried this on an instance with actual generated filesystem dictionaries, mostly just to not mess up the direct tests.  Can do that if needed as a last step.

Reply:
It seems to work as expected: if you put all the dictionaries in, it runs extra queries and merges all the results together. You obviously wouldn't want to do too many for performance reasons, but it doesn't break the system.